_redirects.yml

```diff
@@ -128,7 +128,8 @@
 "/desktop/windows/wsl/":
   # Link used by Docker Desktop to refer users on how to activate WSL 2
   - /go/wsl2/
-
+"/engine/reference/commandline/compose_config/":
+  - /engine/reference/commandline/compose_convert/
```

Comment:
Not the cause of the error, but for these, we usually use the `redirects_from` metadata in the target markdown file. That way the redirects get preserved when the file moves around, which can make it easier to maintain.